rdt-open-reader/src/main/java/io/ona/rdt/activity/PatientRegisterActivity.java

```diff
@@ -209,6 +213,14 @@ public boolean onNavigationItemSelected(MenuItem menuItem) {
                 return true;
             }
         });
+
+        String lastSyncTimestampStr = RDTApplication.getInstance().getContext().allSharedPreferences().getPreference(Constants.Preference.CTS_LATEST_SYNC_TIMESTAMP);
+        if (StringUtils.isNotBlank(lastSyncTimestampStr)) {
+            Date lastSyncDate = new Date(Long.parseLong(lastSyncTimestampStr));
+            String lblSync = getString(R.string.drawer_menu_item_sync) + getString(R.string.last_sync_date_placeholder);
```

Comment:
You can merge `R.string.drawer_menu_item_sync` and `R.string.last_sync_date_placeholder` into into one string.

Reply:
If i merge the placeholder string then the first time placeholder will show with sync label like this `Sync: %1$s`

Reply:
sync label merged with placeholder